### PR TITLE
Feature/2462 web add ability to search all organisations via postcode through find help drop down menu

### PR DESCRIPTION
**Pull request by o-seliuchenko**
https://trello.com/c/C7oOm3kD/2462-web-add-ability-to-search-all-organisations-via-postcode-through-find-help-drop-down-menu

### CODE REVIEW

File: src/js/models/all-organisations/listing.js [hunk 1]
```diff
@@ -242,12 +243,23 @@ function OrgListing (orgsFilter = null, pageSize = 8) {
...
+      location.getPreviouslySetPostcode()
+      .then((result) => {
+        init(result)
+      }, () => {
+        browser.redirect('/500')
+      })
+    }
+  }
+
+  getPostCode()
```
Comment (vincelee888) [inline src/js/models/all-organisations/listing.js:262]: where is the test around this?

File: src/js/pages/locations/app.js [hunk 2]
```diff
@@ -65,6 +65,11 @@ const initFindHelp = function (currentLocation) {
     postcode: document.querySelector('.js-find-help-postcode')
   }
 
+  const allOrganisationsOption = document.createElement('option')
+  allOrganisationsOption.setAttribute('value', 'all-service-providers')
+  allOrganisationsOption.innerText = 'All Organisations'
+  ui.select.appendChild(allOrganisationsOption)
+
```
Comment (vincelee888) [inline src/js/pages/locations/app.js:72]: where is the test around this?

**Review by vincelee888 — CHANGES_REQUESTED** [review]

File: src/js/pages/locations/app.js (hunk 2 above)
Comment (o-seliuchenko) [reply src/js/pages/locations/app.js:72]: I couldn't find any test for testing functionality from this file. I tried to write a test for this piece of code but I didn't have access to the 'document' object (I've got error ReferenceError: document is not defined)

File: src/js/models/all-organisations/listing.js (hunk 1 above)
Comment (o-seliuchenko) [reply src/js/models/all-organisations/listing.js:262]: I've added it in the last commit

File: src/js/pages/locations/app.js (hunk 2 above)
Comment (vincelee888) [reply src/js/pages/locations/app.js:72]: document is the browser, it needs to be wrapped and then stubbed in the test. have a look at browser.js (i think)

Comment (vincelee888) [reply src/js/pages/locations/app.js:72]: ok, just seen that none of the rest is tested either. no worries

**Review by vincelee888 — APPROVED** [review]

**Pull request merged**